_config.yml

```diff
@@ -30,14 +28,6 @@ exclude:
   - LICENSE
 
 defaults:
-  -
-    scope:
-      path: ""
-      type: "pages"
-    values:
-      permalink: "/:basename"
-      layout: "page"
-      sidebar: main
```

Comment:
This is a default for the "Main" button that was decided to be removed from the navbar.